server/modules/surveyRdb/api/surveyRdbApi.js

```diff
@@ -52,6 +52,18 @@ export const init = (app) => {
     }
   )
 
+  app.get('/surveyRdb/:surveyId/view_data_counts', requireRecordListViewPermission, async (req, res, next) => {
```

Comment:
the name is weird, count the number of data into a view?
or count the number of tables by entity as the code says, 

also countsByUuid -> countXXXXXXByUuid

Reply:
ah, maybe we can call it count_view_data_rows or just count_view_data?

Reply:
does it count the number of rows?

Reply:
Yes, it's the count of rows in each entity data view

Reply:
It is quite weird,